test/org/impact2585/frc2016/tests/WheelSystemTest.java

```diff
@@ -37,29 +37,29 @@ public void test() {
 				
 		//tests if the robot isn't moving at the start
 		drivetrain.run();
-		Assert.assertTrue(forwardDistance == 0 && rotationValue == 0);
+		Assert.assertTrue(driveForward == 0 && rotate == 0);
 		
 		//tests deadzone
-		forwardDistance = 0.14;
-		rotationValue = 0.14;
+		driveForward = 0.14;
+		rotate = 0.14;
 		drivetrain.run();
-		Assert.assertTrue(forwardDistance == 0 && rotationValue == 0);
+		Assert.assertTrue(driveForward == 0 && rotate == 0);
 		
 		//tests forward driving
-		forwardDistance = 0.5;
+		driveForward = 0.5;
 		drivetrain.run();
-		Assert.assertTrue(0.5 == forwardDistance && rotationValue == 0);
+		Assert.assertTrue(0.5 == driveForward && rotate == 0);
 		
 		//tests turning
-		rotationValue = 1;
-		forwardDistance = 0;
+		rotate = 1;
+		driveForward = 0;
 		drivetrain.run();
-		Assert.assertTrue(forwardDistance == 0 && rotationValue == 1);
+		Assert.assertTrue(driveForward == 0 && rotate == 1);
 		
 		//tests turning and driving simultaneously
-		rotationValue = 1;
-		forwardDistance = 10;
-		Assert.assertTrue(forwardDistance == 10 && rotationValue == 1);
+		rotate = 1;
+		driveForward = 10;
```

Comment:
Can the input value equal 1?

On Wed, Feb 3, 2016 at 11:17 PM, Michael Murphey <notifications@github.com>
wrote:

> In test/org/impact2585/frc2016/tests/WheelSystemTest.java
> <https://github.com/Impact2585/FRC2016/pull/15#discussion_r51830558>:
>
> >  		
> >  		//tests turning and driving simultaneously
> > -		rotationValue = 1;
> > -		forwardDistance = 10;
> > -		Assert.assertTrue(forwardDistance == 10 && rotationValue == 1);
> > +		rotate = 1;
> > +		driveForward = 10;
>
> This value should be less than 1 since it represents a joystick value
> instead of a distance.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Impact2585/FRC2016/pull/15/files#r51830558>.
>